Changes.md

```diff
@@ -5,6 +5,15 @@
 HEAD
 ---------
 
+- Add `Sidekiq::Job` alias for `Sidekiq::Worker` [#4995]
```

Comment:
- Rename `Sidekiq::Worker` to `Sidekiq::Job` [#4995]
- Alias `Sidekiq::Worker` to `Sidekiq::Job` to keep backward compatibility. [#4995]

Reply:
That way it clear that Job is just the old Worker instead of a new version with same API.